### PR TITLE
[extension-selenium] Add logging for filtering

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 1]
```diff
@@ -181,13 +222,14 @@ void testFindElementsWithChildrenEmptyResults()
...
         Locator childAttributes = new Locator(TestLocatorType.ADDITIONAL_SEARCH,
             VALUE);
-        attributes.addChildLocator(childAttributes);
+        locator.addChildLocator(childAttributes);
         when(testSearch.search(eq(searchContext), any(SearchParameters.class))).thenReturn(webElements);
         when(additionalTestSearch.search(eq(webElement), any(SearchParameters.class))).thenReturn(List.of());
-        List<WebElement> foundElements = searchActions.findElements(searchContext, attributes);
+        List<WebElement> foundElements = searchActions.findElements(searchContext, locator);
         assertEquals(List.of(), foundElements);
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:233]: assertThat(logger.getLoggingEvents, isEmpty()) ?

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 2]
```diff
@@ -131,9 +133,10 @@ void shouldFindElementInSearchContext()
 
         when(testSearch.search(eq(webDriver), any(SearchParameters.class))).thenReturn(List.of(element));
 
-        Locator attributes = new Locator(TestLocatorType.SEARCH, VALUE);
-        assertEquals(Optional.of(element), searchActions.findElement(webDriver, attributes));
+        Locator locator = new Locator(TestLocatorType.SEARCH, VALUE);
+        assertEquals(Optional.of(element), searchActions.findElement(webDriver, locator));
         verifyNoInteractions(uiContext, searchContext);
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:139]: https://github.com/vividus-framework/vividus/pull/1214/files#r540174906

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 3]
```diff
@@ -119,8 +120,9 @@ void shouldReturnFirstElementIfFewFound()
         when(testSearch.search(eq(searchContext), any(SearchParameters.class)))
             .thenReturn(List.of(element1, element2));
         when(uiContext.getSearchContext()).thenReturn(searchContext);
-        Locator attributes = new Locator(TestLocatorType.SEARCH, VALUE);
-        assertEquals(Optional.of(element1), searchActions.findElement(attributes));
+        Locator locator = new Locator(TestLocatorType.SEARCH, VALUE);
+        assertEquals(Optional.of(element1), searchActions.findElement(locator));
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:125]: https://github.com/vividus-framework/vividus/pull/1214/files#r540175487

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 4]
```diff
@@ -97,18 +96,20 @@ void shouldFindElementsUsingLocatorAndUiContext()
...
     @Test
     void shouldReturnEmptyOptionalIfNoElementFound()
     {
         when(testSearch.search(eq(searchContext), any(SearchParameters.class))).thenReturn(List.of());
         when(uiContext.getSearchContext()).thenReturn(searchContext);
-        Locator attributes = new Locator(TestLocatorType.SEARCH, VALUE);
-        assertEquals(Optional.empty(), searchActions.findElement(attributes));
+        Locator locator = new Locator(TestLocatorType.SEARCH, VALUE);
+        assertEquals(Optional.empty(), searchActions.findElement(locator));
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:112]: https://github.com/vividus-framework/vividus/pull/1214/files#r540175487

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 5]
```diff
@@ -97,18 +96,20 @@ void shouldFindElementsUsingLocatorAndUiContext()
         when(testSearch.search(eq(searchContext), any(SearchParameters.class)))
                 .thenReturn(List.of(webElement));
         when(uiContext.getSearchContext()).thenReturn(searchContext);
-        Locator attributes = new Locator(TestLocatorType.SEARCH, VALUE);
-        List<WebElement> foundElements = searchActions.findElements(attributes);
+        Locator locator = new Locator(TestLocatorType.SEARCH, VALUE);
+        List<WebElement> foundElements = searchActions.findElements(locator);
         assertEquals(List.of(webElement), foundElements);
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:102]: https://github.com/vividus-framework/vividus/pull/1214/files#r540175487

File: vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java [hunk 6]
```diff
@@ -86,9 +84,10 @@ void testFindElementsByLocatorSingleAttribute()
     {
         when(testSearch.search(eq(searchContext), any(SearchParameters.class)))
                 .thenReturn(List.of(webElement));
-        Locator attributes = new Locator(TestLocatorType.SEARCH, VALUE);
-        List<WebElement> foundElements = searchActions.findElements(searchContext, attributes);
+        Locator locator = new Locator(TestLocatorType.SEARCH, VALUE);
+        List<WebElement> foundElements = searchActions.findElements(searchContext, locator);
         assertEquals(List.of(webElement), foundElements);
+        assertThat(logger.getLoggingEvents(), equalTo(List.of()));
```
Comment (ikalinin1) [inline vividus-extension-selenium/src/test/java/org/vividus/ui/action/SearchActionsTests.java:90]: https://github.com/vividus-framework/vividus/pull/1214/files#r540175487

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**